roles/openshift_repos/tasks/main.yaml

```diff
@@ -1,13 +1,20 @@
 ---
+- set_fact:
+    deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+    openshift_deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
```

Comment:
Now I see what you're saying. Actually, I want `openshift_deployment_type` to be set globally (i.e. for all depending roles and plays), not just in the `openshift_repos` role. There doesn't seem to be a way other than `set_facts` to do that. Am I missing something?